raco/myrial/parser.py

```diff
@@ -975,8 +975,8 @@ def p_sexpr_boolean_literal(p):
 
     @staticmethod
     def p_sexpr_blob_literal(p):
-        '''sexpr : NULL'''
-        p[0] = sexpr.BlobLiteral(p[1])
+        '''sexpr : EMPTY_BLOB'''
```

Comment:
Um, I don't see where `EMPTY_BLOB` is defined. If it's not a keyword like `NULL`, what is it?